src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+* Added enrich functionality to metric instrumentation
+  ([#1407](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1407)).
+
+  * New overload of `AddAspnetInstrumentation` now accepts a configuration delegate.
+  * The `Enrich` can be used to add additional metric tags.
```

Comment:
I think that we should change tags -> attributes
It is common name in OTel. Tag is the only implementation in .NET.